src/pretix/api/serializers/event.py

```diff
@@ -15,14 +17,73 @@ def to_representation(self, value):
         }
 
 
+class PluginsField(Field):
+
+    def to_representation(self, obj):
+        from pretix.base.plugins import get_all_plugins
+
+        plugins = {
+            p.module for p in get_all_plugins()
+            if not p.name.startswith('.') and getattr(p, 'visible', True) and p.module in obj.get_plugins()
+        }
+
+        return plugins
+
+    def to_internal_value(self, data):
+        from pretix.base.plugins import get_all_plugins
+
+        plugins_available = {
+            p.module for p in get_all_plugins()
+            if not p.name.startswith('.') and getattr(p, 'visible', True)
+        }
+
+        for plugin in data:
+            if plugin not in plugins_available:
+                raise ValidationError(
+                    message=_("Unknown plugin: '%s'."),
+                    params=(plugin,)
+                )
+                break
+
+        plugins = {plugin_name for plugin_name in data}
+
+        return {
+            'plugins': ",".join(plugins)
+        }
+
+
 class EventSerializer(I18nAwareModelSerializer):
-    meta_data = MetaDataField(source='*')
```

Comment:
My bad, removed it initially to avoid having to implement `MetaDataField.to_internal_value()`.

Reply:
I tried to implement `to_internal_value()` but couldn't figure out a clean way of doing it. When I returned a dict it couldn't be applied as value. Any idea as to how to get around this?

Reply:
Since this makes use of many-to-many relations, it seems to be impossible to implement inside the ``Field`` class, it behaves more like the nested resources we have elsewere. I took the liberty of pushing the most elegant version I could come up with (which isn't much) to this PR, feel free to disregard or change: https://github.com/pretix/pretix/pull/756/commits/319621677bcb6e501fb0860ac58ba55d021c1709